projects/adrv9009/a10gx/system_qsys.tcl

```diff
@@ -0,0 +1,9 @@
+
+set dac_fifo_name avl_adrv9009_tx_fifo
+set dac_fifo_address_width 10
```

Comment:
Is this depth sufficient ? It's just 2kS per channel.

Reply:
It's similar to what we have for adrv9371 also. We can probably increase it in the next release